pipeline-steps/common.groovy

```diff
@@ -259,4 +259,37 @@ api_key = ${args.api_key}
   return pyrax_cfg
 }
 
+def prepareConfigs(Map args){
+  conditionalStage(
```

Comment:
I think this would be better off not in its own stage, mainly because other stages call this method, so we'll start having things like this Prepare Configs stage nested inside of the Prepare Deployment stage. I also don't think we'll have a scenario where we would want to run Prepare Deployment but not run Prepare Configs, so it shouldn't be a problem to not have it as a conditional stage.